datasync/chaindatafetcher/kafka/config.go

```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/Shopify/sarama"
 )
 
+const (
+	EventBlockGroup = "blockgroup"
```

Comment:
These constants seem to be used only in `kafka` package, do you have other reason to declare these variables as public ones?

Reply:
Yes, if anybody uses those constants outside after importing the package, I thought they should be public.